DEVELOPING.md

```diff
@@ -1,97 +1,130 @@
 4. Run the app in the normal way
 
-### Beta artefact
+### Beta artifact
 
 1. Follow the [Android repo](https://github.com/embrace-io/embrace-android-sdk3#qa-releases) instructions for creating a beta
 2. Find `rootProject.allprojects.repositories` in `embrace_android/android/build.gradle` and add `maven {url "https://repo.embrace.io/repository/beta"}`
 3. Add `mavenLocal()` to the `repositories` closure in `examples/react-native-test-suite/node_modules/embrace-io/android/build.gradle`
-4. Set the correct `embrace-android-sdk` version in both `examples/react-native-test-suite/node_modules/embrace-io/android/build.gradle`
+4. Set the correct `embrace-android-sdk` version in `examples/react-native-test-suite/node_modules/embrace-io/android/build.gradle`
 5. Run the app in the normal way
 
-## Testing iOS changes locally
+## Testing against new Embrace iOS SDK versions
 
-### Local artefact
+### Local artifact
 
-You can test changes local changes to the iOS SDK by updating the React Native Embrace project's `podspec` and `Podfile` to point to the local copy.
+You can test changes local changes to the iOS SDK by updating the example app's `podspec` and `Podfile` to point to the local copy.
 
 1. In `examples/react-native-test-suite/node_modules/embrace-io/RNEmbrace.podspec`, change the dependency on the iOS SDK to `s.dependency 'EmbraceIO-LOCAL'`
 2. In `examples/react-native-test-suite/ios/Podfile`, add the following line `pod 'EmbraceIO-LOCAL', :path => 'path/to/ios_sdk'`
 3. In `examples/react-native-test-suite/ios`, run the `pod update` command
 
-### Beta artefact
+### Beta artifact
 
 1. Ask the iOS team to publish a beta of the iOS SDK to the `EmbraceIO-DEV` pod
 2. In `examples/react-native-test-suite/node_modules/embrace-io/RNEmbrace.podspec`, change the dependency on the iOS SDK to `s.dependency 'EmbraceIO-DEV'`
 3. In `examples/react-native-test-suite/ios/Podfile`, add the following line `pod 'EmbraceIO-DEV'`
 4. In `examples/react-native-test-suite/ios`, run the `pod update` command
 
-# Releasing the SDK
+## Automated integration testing
+
+Automated integration testing is being actively developed to replace some of the manual testing outlined above, see [here](./integration-tests/README.md) for more details.
 
-0. Bump the Android (SDK + Swazzler)/iOS dependencies to the latest available stable versions
-   0.1. Update the iOS SDK version in `./RNEmbrace.podspec`
-   0.2. Update the Android SDK version in `./android/gradle.properties`
-1. Bump the SDK version according to semver
-2. Run the example app on Android + iOS (in release mode) and confirm that a session is captured & appears in the dashboard with useful info
-3. Create a PR with all these changes and merge to `master`
-4. Release to npm with `yarn publish --tag <your-version-here>`
+## Branching strategy
 
-# Releasing a beta SDK version
+Generally all proposed changes should target the `master` branch and new releases are cut from there after QA. One exception
```

Comment:
@TheMutsi any issue with adopting this approach? Is the current approach to make separate release branches?

Reply:
Yes the idea is making release branches and only put it on master when it is deployed. Unless it is a very small change.
If we follow the release branches approach I think you can cut from maser instead of a tag.